### PR TITLE
Resources

### PR DESCRIPTION
**Pull request by agriffard**
Update bootstrap 3, jQuery, add font awesome. 
Fix SetVersion in StyleTagHelper.
See #421

### CODE REVIEW

**Comment by sebastienros** [comment]
I don't thin we need to embed all the scss for fontaweome, we don't intend to customize it. Contrary to bootstrap where other themes might require the scss and want to override the variables.

File: src/Orchard.Cms.Web/Themes/TheTheme/Views/Layout.cshtml [hunk 1]
```diff
@@ -10,9 +10,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link type="image/x-icon" rel="shortcut icon" href="/TheTheme/Content/favicon.ico" />
 
-    <script asp-name="bootstrap" use-cdn="true" at="Foot"></script>
+    <script asp-name="bootstrap" use-cdn="true" version="3.3.7" at="Foot"></script>
```
Comment (sebastienros) [inline src/Orchard.Cms.Web/Themes/TheTheme/Views/Layout.cshtml]: I think you should only specify 3.3-* or maybe 3-* if it works. This way you don't change the theme when a new bs minor version is shipped.

File: src/Orchard.Cms.Web/Modules/Orchard.Contents/Content/Styles/Contents.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.edit-container {
```
Comment (sebastienros) [inline src/Orchard.Cms.Web/Modules/Orchard.Contents/Content/Styles/Contents.scss]: Ca n you explain the reason for this css and/or some screenshots?

File: src/Orchard.Cms.Web/Modules/Orchard.Resources/Assets.json [hunk 3]
```diff
@@ -8,5 +8,9 @@
       "Assets/Bootstrap/Js/bootstrap.js"
     ],
     "output": "Content/Scripts/bootstrap.js"
+  },
+  {
+    "inputs": [ "Assets/font-awesome/scss/font-awesome.scss" ],
+    "output": "Content/Styles/font-awesome.css"
```
Comment (sebastienros) [inline src/Orchard.Cms.Web/Modules/Orchard.Resources/Assets.json]: In the end you can just copy the official css files in this folder. They shouldn't be generated by our pipeline.

File: src/Orchard.Cms.Web/Themes/TheTheme/Views/Layout.cshtml (hunk 1 above)
Comment (agriffard) [inline src/Orchard.Cms.Web/Themes/TheTheme/Views/Layout.cshtml]: 3.3-*, 3.3.*, 3.* don't work.

**Comment by sebastienros** [comment]
@agriffard Fixed the version filter, please review and merge

**Pull request merged**